pod/perlguts.pod

```diff
@@ -158,30 +158,34 @@ To access the actual value that an SV points to, you can use the macros:
     SvIV(SV*)
     SvUV(SV*)
     SvNV(SV*)
-    SvPV(SV*, STRLEN len)
-    SvPV_nolen(SV*)
+    SvPVbyte(SV*, STRLEN len)
```

Comment:
I'm not sure if ignoring `SvPV`/`SvPV_nolen` entirely here is the right solution.

Reply:
It’s discussed elsewhere, at least., but yeah.

CDB_File had a bug for years where the SV’s internal state dictated how it appeared in the CDB output. I noticed the same problem recently in URI::XSEscape and Net::Curl, and I began to think that pretty much any XS that outputs to something besides Perl probably has more use for the byte/utf8 variants of SvPV, so that a string will translate consistently to C regardless of whether it’s bytes/Latin-1 or UTF-8 internally.

Reply:
At the very least, people should be advised that the only correct use of SvPV is in conjunction with knowledge of the UTF8 bit.

Reply:
I also like to see them kept, if not only to be complete. Just add a warning if desired.
I'm with Grinnz here

Reply:
I’m resolving this, as I think I’ve addressed the concern.